docs/guide/bundles.md

```diff
@@ -24,59 +24,130 @@ To learn more about organizing your eZ Platform project, see [Best Practices](be
-```
-
-#### PlatformUIBundle
-
-[PlatformUIBundle](https://github.com/ezsystems/PlatformUIBundle) provides the main editing and back-end interface for eZ Platform.
-
-#### ezplatform-solr-search-engine
-
-[ezplatform-solr-search-engine](https://github.com/ezsystems/ezplatform-solr-search-engine)
-is the [Solr-powered](http://lucene.apache.org/solr/) search handler for eZ Platform.
-
-#### Repository Forms
-
-[Repository Forms](http://github.com/ezsystems/repository-forms) is a bundle which provides form-based interaction for the Repository Value objects.
-
-It is currently used by:
-
-- `ezsystems/platform-ui-bundle` for most management interfaces: Sections, Content Types, Roles, Policies, etc.
-- `ezsystems/ezpublish-kernel` for user registration and user generated content
-
-!!! enterprise
-
-    #### StudioUIBundle
-
-    StudioUIBundle contains the Studio editing interface
-    provided in eZ Platform Enterprise Edition.
-
-    #### Landing Page Field Type Bundle
-
-    EzLandingPageFieldTypeBundle provides
-    the Landing Page that is at the heart of StudioUI.
+The eZ Platform directory contains multiple sub-directories.
+Each sub-directory is dedicated to a specific part of the system and contains a collection of logically related files.
 
 !!! tip
 
     You can see the bundles that are automatically installed with eZ Platform in [composer.json](https://github.com/ezsystems/ezplatform/blob/master/composer.json).
+
+The following tables give an overview of the main eZ Platform directories.
+
+### Core bundles
+
+|Bundle|Description|
+|---------|-----------|
+|[ezplatform](https://github.com/ezsystems/ezplatform)|meta repository that pulls in all dependencies for clean distribution of eZ Platform|
+|[ezpublish-kernel](https://github.com/ezsystems/ezpublish-kernel)|contains the core of the whole eZ Platform application e.g. EzPublishCoreBundle|
+|[repository-forms](https://github.com/ezsystems/repository-forms)|provides form-based interaction for the Repository Value objects|
+|[ezplatform-solr-search-engine](https://github.com/ezsystems/ezplatform-solr-search-engine)|[Solr-powered](http://lucene.apache.org/solr/) search handler for eZ Platform|
+|[ez-support-tools](https://github.com/ezsystems/ez-support-tools)|provides functionality for system information|
+|[ezplatform-http-cache](https://github.com/ezsystems/ezplatform-http-cache)|HTTP cache handling for eZ Platform, using multi tagging (incl Varnish xkey)|
+|[ezplatform-design-engine](https://github.com/ezsystems/ezplatform-design-engine)|design fallback system for eZ Platform similar to legacy design fallback system|
+|[ezplatform-standard-design](https://github.com/ezsystems/ezplatform-standard-design)|defines standard Design and Theme to be handled by ezplatform-design-engine|
+|[ezplatform-cron](https://github.com/ezsystems/ezplatform-cron)|exposes cron/cron package for use in eZ Platform (or just plain Symfony) via a simple command `ezplatform:cron:run`|
```

Comment:
Might be only 2.2 and up, but technally there is no reason why it needs to be, reason could be non technical:  avoiding confusing 1.x EE users who already have the former (99% same) ezstudio-cron.
 
ref: https://github.com/ezsystems/ezplatform-cron/commit/40fed0d16270d002f1edf578f82e56498c5914da